addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3413,15 +3413,21 @@ export class OdooEditor extends EventTarget {
             } else if (['insertText', 'insertCompositionText'].includes(ev.inputType)) {
                 // insertCompositionText, courtesy of Samsung keyboard.
                 const selection = this.document.getSelection();
-                // Detect that text was selected and change behavior only if it is the case,
-                // since it is the only text insertion case that may cause problems.
-                const wasTextSelected = anchorNodeOid !== focusNodeOid || anchorOffset !== focusOffset;
+                // Detect that text was selected accross nodes and change the
+                // behavior only if it is the case, since it is the only text
+                // insertion case that may cause problems.
+                const anchorNode = this.idFind(anchorNodeOid);
+                const wasSelectionSpanningMultipleNodes =
+                    anchorNodeOid !== focusNodeOid ||
```

Comment:
... but actually this suggests that multiple text nodes could in fact do? Did I misunderstand then?

Should we enter the condition if we have the word `"odoo"` split in two (`"od"``"oo"`)?